v2/yarpcprotobuf/protoc-gen-yarpc-go/internal/tests/src/stream/stream.go

```diff
@@ -34,17 +36,28 @@ func NewServer() streampb.HelloYARPCServer {
 }
 
 func (h *helloServer) In(req *streampb.HelloRequest, s streampb.HelloInYARPCStreamServer) error {
-	resp := fmt.Sprintf("Received %q", req.GetGreeting())
-	return s.Send(&streampb.HelloResponse{Response: resp})
+	for i := 0; i < len(req.GetGreeting()); i++ {
```

Comment:
nit: may want to add a comment that we're sending the length of the greeting, since it's a bit unusual

Reply:
Good call, I added comments to all of the stream methods for clarity.